UltraStar Play/Assets/Editor/BuildCommand.cs

```diff
@@ -0,0 +1,108 @@
+﻿using UnityEditor;
```

Comment:
This file has mixed tabs and spaces (I'm surprised that Codacy didn't pick up on it?). Should be space-only.

Reply:
sorry, my fault. Neither the compiler nor most modern IDEs care about tabs or spaces at line beginning, so I didn't notice the difference. I'll update the PR.

Reply:
Looks good now.